Add `strict=True` explicit flag to Executorch invocations to torch.export.export

gmagogsfm opened the pull request:
Differential Revision: D69068797

gmagogsfm commented:
@pytorchbot label "topic: not user facing"

iseeyuan reviewed (approved):
LGTM. Would we moved to strict=False as suggested?

gmagogsfm committed:
executorch (#8190)

Summary: Pull Request resolved: https://github.com/pytorch/executorch/pull/8190

Reviewed By: avikchaudhuri, iseeyuan

Differential Revision: D69068797
